src/widgets/category/category-widget-model.js

```diff
@@ -136,10 +136,45 @@ module.exports = WidgetModel.extend({
     }
   },
 
+  sizeAcceptedCatsInData: function () {
```

Comment:
I'm on doubt of these methods, should they be here or in CartoDB.js?

@viddo @alonsogarciapablo